GenomeFileUtil.spec

```diff
@@ -236,9 +236,6 @@ module GenomeFileUtil {
     funcdef export_genome_as_gff(ExportParams params)
                 returns (ExportOutput output) authentication required;
 
-    funcdef export_genome_features_protein_to_fasta(ExportParams params)
```

Comment:
I assume no other model is using this function? We should probably mention this in RELEASE note.

Reply:
They can't be, the return var is `output` which is never initialized.

+1 on release notes though

Reply:
updated release notes 👍